### DISCUSSION TITLE
Quest for ES6 module sample code

### DISCUSSION THREAD

**Discussion by dorcatty** [kurokida/jsQuestPlus#7] [General]
Dear author,

Thanks for the updated version of jsQuest, now we have jsQuestPlus!
however, as a developer, we need to integrate jsQuestPlus into our environment which is ES6 based. 
it seems that there's no sample code was given to explain how to run jsQuestPlus wish javascript ES6.
but this feature is given by jsQuest: jsQUEST/test folder, which is useful. 
Could you please also provide a piece of sample code of using jsQuestPlus with javascript ES6, just as jsQuest ?

thanks a lot

**Comment by tpronk** [comment]
Hi dorcatty, thanks for your question! If @kurokida is OK with it, I'd like to try provide the sample code you're looking for. Just to check: you're looking for a jsQuestPlus version of this sample right? https://github.com/kurokida/jsQUEST/blob/main/test/jsQUEST_test.js

**Reply by dorcatty** [reply]
exactly, that is the ES6 use case of jsQuest, i was looking for jsQuestPlus version, but it is not given at the moment. 
with jsQuestPlus, only examples in script tag in html format. thx a lot

**Reply by tpronk** [reply]
No prob. I made [an example](https://github.com/tpronk/jsQuestPlus/blob/main/test/jsQuestPlus_test.js) in my fork of jsQuestPlus. Hope it helps you out!

**Comment by dorcatty** [comment]
cool, nice, not i can see the ES6 version, and get the general idea. nice
job.
Best

**Comment by kurokida** [comment]
Hi, @dorcatty , thank you for using our library. Have you solved all your problems? If they remain, please don't hesitate to ask us.

@tpronk , thank you for a quick response!
